Document/0x05a-Platform-Overview.md

```diff
@@ -262,48 +262,138 @@ We recommend that you test both the APK with and without the additional modules,
 
 ### Android Manifest
 
-Every app has an Android Manifest file, which embeds content in binary XML format. The standard name of this file is AndroidManifest.xml. It is located in the root directory of the app's Android Package Kit (APK) file.
+Every Android app contains an `AndroidManifest.xml` file in the root of its APK, written in binary XML format. This file defines the app's structure and key properties used by the Android operating system during installation and runtime.
+
+Security-relevant elements include:
+
+- **Permissions:** Declares required permissions using `<uses-permission>` such as access to the internet, camera, storage, location, or contacts. These define the app's access boundaries and should follow the principle of least privilege. Custom permissions can be defined using `<permission>` and should include a proper `protectionLevel` such as `signature` or `dangerous` to avoid being misused by other apps.
+- **Components:** The manifest lists all [app components](#app-components) declared in the app serving as entry points. They can be exposed to other apps (via intent filters or the `exported` attribute) so they are critical to determine how an attacker might interact with the app. The main component types are:
+    - **Activities:** define user interface screens. If associated with intent filters and not properly restricted, they may be invoked by other apps.
```

Comment:
exported=true without intent filter would also work

Reply:
Maybe just remove it from all of these? Because they're all just the same. So stick to defining them without saying each time that anything exposed can be attacked?